### ISSUE TITLE
Component mode failing with recursive errors on init

### ISSUE THREAD

**Issue by jaybo** [vue-youtube/vue-youtube#2]
```
...
   <youtube-iframe video-id="dQw4w9WgXcQ" />
...
    </q-card>
</template>

<script>
...
import { YoutubeIframe } from '@vue-youtube/component';
```
The import line is grayed out in vscode with the error:
`'YoutubeIframe' is declared but its value is never read.ts(6133)`

The recursive errors include:
```
Property "trim" was accessed during render but is not defined on instance. at <DocEdit>
at <BaseTransition>
at <Transition>
at <QDialog>
...
Property "replace" was accessed during render but is not defined on instance. at <DocEdit>
at <BaseTransition>
at <Transition>
at <QDialog>
...

Property "toString" was accessed during render but is not defined on instance. at <DocEdit>
at <BaseTransition>
at <Transition>
at <QDialog>
```
One other note:  During yarn install, I got some message about needing to select which version to install and the only choice was 0.0.1.  Unusual...

**Comment by Techassi** [comment]
Mhm this seems weird. Will look into it!

**Comment by Techassi** [comment]
Sorry for the delayed answer by the way. I tested the following code and this works without any issues:

```vue
<template>
  <youtube-iframe video-id="dQw4w9WgXcQ" />
</template>

<script setup lang="ts">
import { YoutubeIframe } from '@vue-youtube/component';
</script>
```

About the version selection during install: unusual indeed. Maybe some issue with the version I tagged; I had a small typo.

**EDIT:** The errors above seem unrelated to this plugin. This plugin doesn't use any of these custom elements / components. I will also move this to the main repo at [vue-youtube/vue-youtube](https://github.com/vue-youtube/vue-youtube).
